spotifyutils/__init__.py

```diff
@@ -49,8 +46,49 @@ def cli(input_args=None):
     )
 
     args = parser.parse_args(input_args)
-    main(**vars(args))
+    configuration(**vars(args))
+
+def configuration(**kwargs):
+    """ Interpret CLI args and create configuration """
```

Comment:
This docstring is not very informative. What does it do? Does it return a value? Does it have side effects?